Makefile

```diff
@@ -168,6 +168,10 @@ auraed-test: musl proto
 auraed-test-all: musl proto
 	@sudo -E $(cargo) test --target $(uname_m)-unknown-linux-musl -p auraed -- --include-ignored
 
+.PHONY: auraed-test-watch
+auraed-test-watch: musl proto # Use cargo-watch to continuously run a test (e.g. make auared-test-watch name=path::to::test)
+	@sudo -E $(cargo) watch -- $(cargo) test --target $(uname_m)-unknown-linux-musl -p auraed $(name)
```

Comment:
should we add `--include-ignored` here so we can also use this for all our ignored tests?

Reply:
Seems like a good idea. I'll update. Grab whatever you need though from this PR for yours